src/index.html

```diff
@@ -1,12 +1,332 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="img-logo">
+        <a href="#" class="img-logo__main-page">
+          <img
+          src="images/logo.png"
+          alt="logo-moyo"
+          width="40px"
+          height="40px"
+          >
+        </a>
+      </div>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__list-item navigation__list-item--left">
```

Comment:
```suggestion
          <li class="navigation__list-item">
```
Просто використай first-child для navigation__list-item